nerfstudio/scripts/exporter.py

```diff
@@ -497,7 +504,26 @@ def main(self) -> None:
         map_to_tensors = {}
 
         with torch.no_grad():
-            positions = model.means.cpu().numpy()
+            if self.save_world_frame:
+                assert isinstance(pipeline.datamanager, FullImageDatamanager)
+                dataparser_outputs = pipeline.datamanager.train_dataparser_outputs
+                dataparser_scale = dataparser_outputs.dataparser_scale
+                dataparser_transform = dataparser_outputs.dataparser_transform.numpy(force=True)
+
+                output_scale = 1 / dataparser_scale
+                output_transform = np.zeros((3, 4))
+                output_transform[:3, :3] = dataparser_transform[:3, :3].T
```

Comment:
pretty please don't do transform math w/out at least comments, this sort of code is 110% likely to put a future reader in transform hell

also pretty please use ` pipeline.datamanager.train_dataparser_outputs.transform_poses_to_original_space()` because 
(1) that's what's used elsewhere in this file
(2) using that function ensures future refactors won't break things, and most past nerfstudio refactors have indeed broken lots of things